src/termdict/mod.rs

```diff
@@ -20,37 +20,18 @@ as `u64`.
 A second datastructure makes it possible to access a [`TermInfo`](../postings/struct.TermInfo.html).
 */
 
-use tantivy_fst::automaton::AlwaysMatch;
-
 mod fst_termdict;
 use fst_termdict as termdict;
 
 mod merger;
 
-#[cfg(test)]
-mod tests;
+pub use self::merger::TermMerger;
+pub use self::termdict::TermDictionary;
+pub use self::termdict::TermDictionaryBuilder;
+pub use self::termdict::TermStreamer;
 
 /// Position of the term in the sorted list of terms.
 pub type TermOrdinal = u64;
 
-/// The term dictionary contains all of the terms in
```

Comment:
Do we have comments on the TermDictionary, etc. themselves?

Reply:
Yup :)